### DISCUSSION TITLE
Discussion on inclusion of postject in the Node.js binary

### DISCUSSION THREAD

**Discussion by RaisinTen** [nodejs/single-executable#53] [General]
See @mhdawson's comment in https://github.com/nodejs/node/pull/45066#issuecomment-1286182691.

Doodle link: https://doodle.com/meeting/participate/id/avoKgV0b.
Can everybody please fill in their availabilities? I'll schedule the meeting accordingly.

cc @nodejs/single-executable

**Comment by jviotti** [comment]
Once the date selection is finalized, can we broadcast the invite in some way other than Doodle too? I think I never got the invite last time we used Doodle 😓

**Reply by RaisinTen** [reply]
Sure, once the date is finalized, do I create a new issue with the time, agenda items and the meeting link and ping everyone or is there a better way?

**Reply by jviotti** [reply]
Sounds great, thanks! :)

**Reply by ovflowd** [reply]
Same for me, let's use something different :D

**Reply by RaisinTen** [reply]
Done, I have booked the event on doodle based on the date with the majority of participants.

(Note that the time in the picture is in IST, so don't get confused!)

<img width="684" alt="Screen Shot 2022-10-28 at 12 55 45 PM" src="https://user-images.githubusercontent.com/42526976/198531819-0f905cb7-adee-48d0-b762-b47b5334a5ee.png">

This should've sent y'all an email. I have also created an issue for better visibility - https://github.com/nodejs/single-executable/issues/54.
